packages/forms-web-app/src/pages/projects/project-information/includes/contact.njk

```diff
@@ -0,0 +1,33 @@
+<section>
+	<h2 class="govuk-heading-m">Contact us</h2>
+
+	<h3 class="govuk-heading-s">
+		Telephone
+	</h3>
+	<p>If you have an interested party number, have it with you when you call.
```

Comment:
<p> needs to have a closed tag '</p>' and ideally have the 'govuk-body' class added to it